### PR TITLE
add iaviewer

### PR DESCRIPTION
**Pull request by zhshy11**
add iaviewer
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: cmd/exchaind/iaviewer.go [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+	KeyStaking      = "s/k:staking/"
+	KeySlashing     = "s/k:slashing/"
+	KeyTokenPair    = "s/k:token_pair/"
+	KeyUpgrade      = "s/k:upgrade/"
+	KeyFarm         = "s/k:farm/"
+	KeyOrder        = "s/k:order/"
+	KeyDex          = "s/k:dex/"
+	KeyAmmswap      = "s/k:ammswap/"
+	KeyEvidence     = "s/k:evidence/"
+	KeyLok          = "s/k:lock/"
```
Comment (xiangjianmeng) [inline cmd/exchaind/iaviewer.go]: KeyLock

File: cmd/exchaind/iaviewer.go [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+	acctypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	distypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/okex/exchain/x/distribution/types"
+)
+
+const (
+	KeyDistribution = "s/k:distribution/"
```
Comment (xiangjianmeng) [inline cmd/exchaind/iaviewer.go:36]: these keys can be replaced by `keys := sdk.NewKVStoreKeys` in app.go

File: cmd/exchaind/iaviewer.go (hunk 1 above)
Comment (zhshy11) [reply cmd/exchaind/iaviewer.go]: done

File: cmd/exchaind/iaviewer.go (hunk 2 above)
Comment (zhshy11) [reply cmd/exchaind/iaviewer.go:36]: done

**Review by xiangjianmeng — APPROVED** [review]

**Pull request merged**